core/pom.xml

```diff
@@ -218,6 +218,16 @@
                 </executions>
             </plugin>
             -->
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
That worked. I used a property that's only defined in the noparameters pom. Thanks